pages/common/unzstd.md

```diff
@@ -0,0 +1,16 @@
+# unzstd
+
+> Decompress files with Zstandard compression.
+> More information: <https://github.com/facebook/zstd>.
+
+- Decompress files into the current directory:
```

Comment:
Oh, the command doesn't extract in the current directory. If you use this
```bash
unzstd ~/Downloads/pacman-6.0.1-7-x86_64.pkg.tar.zst
```
The tar is placed into the `Downloads` directory. So,
```suggestion
- Decompress files:
```

Reply:
Good catch!